db/migrate/20160114153948_delete_cloud_network_id_from_network_ports.rb

```diff
@@ -1,6 +1,11 @@
 class DeleteCloudNetworkIdFromNetworkPorts < ActiveRecord::Migration
-  def change
+  def up
     remove_index :network_ports, :column => :cloud_network_id
```

Comment:
@chrisarcand right. Hm best if index handling could be a part of add/remove_column, then it could work with change. :-)